pages/Messages/Validation.md

```diff
@@ -0,0 +1,173 @@
+---
+name: Validation
+route: /Messages/Validation
+menu: Messages
+---
+
+# Message Validation
+
+All messages read from an archive should be validated to guarantee the message's authenticity, authorization and legality.
+Messages should be validated before being written to an archive file, however clients should not rely on this being the case and should perform their own validation on messages in the interest of protection against malicious actors.
+
+Validation for a message can be defined as a collection of independent steps which may be run in parallel depending on implementation, however all steps are required to pass for a message to be considered valid.
+Certain validation steps are applicable to all messages types and will be referred to as "General Validation" steps in this document.
+Other validation steps, referred to as "Type Specific Validation" steps, may only be applied to certain message types.
+Additionally, some validation steps are entirely optional and may be applied depending on user choice and legal jurisdiction.
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+1. Describe the process of validating messages for inclusion a batch
+
+## Assumptions
+* All assumptions from [DSNP Messages](/Messages/Overview)
+* All assumptions from [DSNP Signatures](/Messages/Signatures)
+
+## General Message Validation
+
+In general, all implementations should validate all announcements for correctness against the [DSNP message schema](/Messages/Overview) and authenticity using [their provided signatures](/Messages/Signatures).
+Additionally, all client implementations should validate content for correctness against the [Activity Pub schema](https://www.w3.org/TR/activitypub/) and authenticity using the provided content hash.
+However, archivist implementations may choose to skip content validation steps in the interest of performance.
+
+### Announcement Correctness
+
+Validation of announcement correctness will vary depending on which fields are present on a particular DSNP message type.
+In general, correctness validation will consist of confirming that all necessary fields are present and that values in each field use a format appropriate for the field type.
+
+#### Encrypted Messages
```

Comment:
I was thinking it would be relevant to distinguish encrypted vs. non-encrypted messages here because we're under the "Announcement Correctness" subsection. The "Announcement Authenticity" subsection below this one goes into more detail regarding signatures.